src/components/SideBar/SideBar.tsx

```diff
@@ -113,6 +121,14 @@ export interface SideBarProps {
    * Optional object for the styles of the SideBar
    */
   style?: CSSProperties
+  /**
+   * Optional background item active
+   */
+  backgroundItemActive?: string
```

Comment:
Well, i think we can move this and the below, inside the subItems type like i suggested on the top